packages/eds-core-react/src/components/Autocomplete/Option.tsx

```diff
@@ -41,19 +41,21 @@ const StyledListItem = styled.li<StyledListItemType>(
   },
 )
 
-const Label = styled.span<{ $multiline: boolean }>(({ theme, $multiline }) => {
-  return css`
-    ${spacingsTemplate(theme.entities.label.spacings)}
-    text-overflow: ellipsis;
-    white-space: ${$multiline ? 'normal' : 'nowrap'};
-    overflow: ${$multiline ? 'initial' : 'hidden'};
-    overflow-wrap: anywhere;
-    /* hack to fix clipping issue in firefox (#3170) */
-    @supports (-moz-appearance: none) {
-      overflow: ${$multiline ? 'initial' : 'clip'};
-    }
-  `
-})
+export const AutocompleteOptionLabel = styled.span<{ $multiline: boolean }>(
```

Comment:
Would it be simpler to name it Label? What is the downside of having a generic name?

Reply:
It would be simpler, and Label was the original name when it wasn't exported. But I wanted to follow the naming of the other exported variables in the file - like  "AutocompleteOption". And since we also have a different Label component I thought it was best to differentiate the two.